lib/src/disposable_manager/disposable_manager_vm.dart

```diff
@@ -14,7 +14,7 @@
 
```

Comment:
We will actually need to create a `disposable_manager_browser.dart` in the next iteration of this when we implement the new browser methods on the manager -- the manager will follow the same extending pattern as `Disposable`. So naming this file like this was a bit of forward thinking, but I'll leave it off for now until we're at that point to eliminate confusion.